pkg/provider/provider_helpers.go

```diff
@@ -187,3 +201,29 @@ func GetAccessTokenWithRefreshToken(
 	}
 	return result.AccessToken, nil
 }
+
+func envNameFieldDescription(description, envName string) string {
+	return fmt.Sprintf("%s Can also be sourced from the `%s` environment variable.", description, envName)
+}
+
+// TODO(SNOW-1752043 - next pr): move to common with resources
```

Comment:
next Pr will touch again the same files, because it will have to be exported after moved to another package, maybe let's move it already. To not alter all the usages in the resources package, you can temporarily create unexported method there, and removal of it can be handled in the subsequent PR

Reply:
Done.